lib/msal-react/docs/class-components.md

```diff
@@ -43,14 +48,16 @@ class App extends React.Component {
             <MsalProvider instance={pca}>
                 <AuthenticatedTemplate>
                     <span>This will only render for authenticated users</span>
-                </ AuthenticatedTemplate>
+                </AuthenticatedTemplate>
                 <UnauthenticatedTemplate>
                     <span>This will only render for unauthenticated users</span>
-                </ UnauthenticatedTemplate>
-                <MsalAuthenticationTemplate interactionType={InteractionType.Popup}>
+                </UnauthenticatedTemplate>
+                <MsalAuthenticationTemplate
+                    interactionType={InteractionType.Popup}
+                >
```

Comment:
This change does not seem correct. And many other changes in this PR are introducing new lines and spaces where not needed. Can you fix that?

Reply:
@neha-bhargava I explained this in the description for this PR.